src/main/resources/application.yaml

```diff
@@ -12,6 +12,10 @@ spring:
 
 bulk-scan-processor-url: ${BULK_SCAN_PROCESSOR_URL:}
 
+queue:
+  connection-string: ${QUEUE_CONNECTION_STRING:}
+  read-interval: ${QUEUE_READ_INTERVAL:30000} # in ms
```

Comment:
Won't we need to rename all the connection string variables if we need to add other queues?

Reply:
**IF** we do, then yes :)